### PR TITLE
Dockerfile to run the application in container

### PR DESCRIPTION
**Pull request by mloyanich**
With Dockerfile we will be able to run API application in container and deploy it to Heroku

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+WORKDIR /app
+
+RUN useradd -m -r ninja && \
+    chown ninja /app
+
+COPY requirements.txt requirements.txt
+RUN pip3 install --no-cache-dir -r requirements.txt
+
+COPY . .
```
Comment (pachkovska) [inline Dockerfile:11]: this line seems to include copied requirements.txt on line 8. you could just get rid of line 8, keep this one and just install after.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Install prerequisites
+
+```bash
+pip install -r requirements.txt
+```
+
+Run the application
+
+```basjj
```
Comment (pachkovska) [inline README.md:21]: typo ?

File: Dockerfile (hunk 1 above)
Comment (mloyanich) [reply Dockerfile:11]: I was pondering about that. will update

**Review by pachkovska — APPROVED** [review]
LGTM!

**Pull request merged**